File: problem_thirty_two.js
var inp = 55
var inp2 = 100
var inp3 = 6
function Myfunction(lower,upper,divi){
    for(var i = lower; i < upper; i++){
        val = i % divi
        if (val == 0){
            console.log(i)
        }
    }
}
Myfunction(inp,inp2,inp3)

File: problem_thirty_two.py
inp = int(input("Enter lower number:"))
inp1 = int(input("Enter upper number:"))
inp2 = int(input("Enter divison number:"))
def MyFunction(lower,upper,divi):
    for i in range(lower,upper,1):
        mod = i % divi
        if mod == 0:
            print(i)

MyFunction(inp,inp1,inp2)    

